src/Hypergrid.js

```diff
@@ -3365,8 +3380,58 @@ Hypergrid.prototype = {
         }
 
         return result;
-    },
+    }
 };
+function findOrCreateContainer(boundingRect) {
+    boundingRect = boundingRect || {};
+    boundingRect.width = boundingRect.width || '300px';
+    boundingRect.height = boundingRect.height || '300px';
+    boundingRect.position = boundingRect.position || 'relative';
+
+    var div,
+        defaultID = 'hypergrid';
+    div = document.getElementById(defaultID);
+    if (div){
+        return div;
+    }
+    div = document.createElement('div');
+    div.setAttribute('id', defaultID);
+    div.style.width = boundingRect.width;
+    div.style.height = boundingRect.height;
```

Comment:
add need `div.style.left || 0 = boundingRect.left` and `div.style.top = boundingRect.top`